app/controllers/your-psp/your-psp-tasks.service.test.js

```diff
@@ -0,0 +1,153 @@
+const { expect } = require('chai')
+const { getCurrentCredential } = require('../../utils/credentials')
+const { getStripeTaskList, stripeTaskListIsComplete } = require('./your-psp-tasks.service')
+const gatewayAccountFixtures = require('../../../test/fixtures/gateway-account.fixtures')
+
+describe('Progress of Stripe PSP tasklist page', () => {
```

Comment:
One more test can be added for the error scenario (i.e., when payment provider is not Stripe)

Reply:
Done